src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -34,9 +34,11 @@ const DragDropcontext: FC<dragDropProps> = ({
   const [isTaskOnDrag, setIsTaskOnDrag] = useState<boolean>(false);
 
   useEffect(() => {
-    const oldTasksIds = taskList.map((Task: task) => Task.id);
-    const newTasks = unAssignedTasks.filter((Task: task) => !oldTasksIds.includes(Task.id));
-    const newTaskList = [...taskList, ...newTasks];
+    const newTaskList = ():task[] => {
```

Comment:
We should place it in the **helperFunctions** folder  
If possible lets have better name such that by looking at it we should know where we will be using it

Reply:
Any suggestions for the function name @Pavangbhat.